packages/devtools_app/lib/src/inspector/inspector_service.dart

```diff
@@ -875,30 +844,34 @@ class ObjectGroup implements Disposable {
   }
 
   Future<List<RemoteDiagnosticsNode>> parseDiagnosticsNodesObservatory(
-      FutureOr<InstanceRef> instanceRefFuture,
-      RemoteDiagnosticsNode parent) async {
+    FutureOr<InstanceRef> instanceRefFuture,
+    RemoteDiagnosticsNode parent,
+    bool isProperty,
+  ) async {
     if (disposed || instanceRefFuture == null) return [];
     final instanceRef = await instanceRefFuture;
     if (disposed || instanceRefFuture == null) return [];
     return parseDiagnosticsNodesHelper(
-        await instanceRefToJson(instanceRef), parent);
+        await instanceRefToJson(instanceRef), parent, isProperty);
```

Comment:
nit: trailing comma, here and the couple spots below line 859, 871

Reply:
done